Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/EntryDelegate.cpp

```diff
@@ -148,7 +148,6 @@ namespace AzToolsFramework
                 {
                     QIcon icon;
                     icon.addFile(path, size, QIcon::Normal, QIcon::Off);
-                    AZ_Assert(!icon.isNull(), "Asset Browser Icon not found for file '%s'", path.toUtf8().constData());
```

Comment:
Should we make this an `AZ_Warning` at least so we can still capture if this case is happening?

Reply:
Sure, I can change it to an AZ_Warning. It didn't seem like the correct behavior to break if the icon isn't found, since the functionality still works but there's just a blank icon for that item.